applications_tests/dem_2d/circle_with_floating_walls.mpirun=2.output

```diff
@@ -7,20 +7,21 @@ Starting simulation with Lethe/DEM on 2 processors
 18 particles of type 0 were inserted, 0 particles of type 0 remaining
 ***************************************************************** 
 id, type, dp, x, y, z 
-0 0 0.00500 -0.0113 -0.0964
+0 0 0.00500 -0.0114 -0.0964
```

Comment:
What's the reason for the results changing?

Reply:
The insertion points are a little different in parallel